src/category_theory/sites/sheaf_of_types.lean

```diff
@@ -85,16 +85,16 @@ which can be convenient.
 
 -/
 
-universes v u
+universes v u₁ u₂
 namespace category_theory
 
 open opposite category_theory category limits sieve classical
 
 namespace presieve
 
-variables {C : Type u} [category.{v} C]
+variables {C : Type u₁} [category.{v} C] {D : Type u₂} [category.{v} D] (F : D ⥤ C)
```

Comment:
Since the variables `D` and `F` are only used in a very small section 200 lines below, I think it would be easier to comprehend if they were declared only in that section.